src/Upbit/Utils.jl

```diff
@@ -0,0 +1,35 @@
+# Upbit/Utils
+
+function Serde.deser(::Type{<:UpbitData}, ::Type{<:Maybe{NanoDate}}, x::Int64)::NanoDate 
+    return unixnanos2nanodate(x*1e6)
+end
+
+function Serde.deser(::Type{<:UpbitData}, ::Type{<:Maybe{NanoDate}}, x::String)::NanoDate 
+    return NanoDate(x)
+end
+
+function Serde.deser(::Type{<:UpbitData}, ::Type{<:Maybe{Date}}, x::String)::Date
```

Comment:
Can you provide examples of methods where different types of time are encountered? Checking by length is unsafe

Reply:
```julia
function Serde.deser(::Type{<:UpbitData}, ::Type{<:Maybe{Date}}, x::String)::Date
    return if occursin(r"^\d{4}-\d{2}-\d{2}$", x)
        Date(x, "yyyy-mm-dd")
    elseif occursin(r"^\d{8}$", x)
        Date(x, "yyyymmdd")
    else
        throw("Unrecognized data format in Upbit: $x")
    end
end
```

Reply:
I suggest abstracting the specific time handling into a module

Reply:
I suggest keeping the specific time handling inside the module

Reply:
Maybe it's better to just `Date(replace(x, "-" => ""), "yyyymmdd")`